pkg/ddc/base/syncs.go

```diff
@@ -54,11 +56,17 @@ func (t *TemplateEngine) Sync(ctx cruntime.ReconcileRequestContext) (err error)
 		return
 	}
 
-	// 4. Update dataset
-	_, err = t.Implement.UpdateOnUFSChange()
+	// 4. Update dataset mount point
+	updateReady, err := t.Implement.UpdateOnUFSChange()
```

Comment:
How about to add `ShouldUpdateOnUFSChange` ?Make it as the same style as `ShouldSetupWorkers`